README.md

```
@@ -187,57 +175,132 @@ _말 한 줄이 아니라, 잠깐 머물 수 있는 순간으로 만들고자 
 
-* 🔗 링크 하나로
+- 🔗 링크 하나로
   **전달하고, 다시 꺼내볼 수 있는 기억**
 
 ---
 
-## 🛠️ 기술 스택
+## � 프로젝트 구조
 
-<img src="https://img.shields.io/badge/React-61DAFB?style=flat-square&logo=react&logoColor=black"/> <img src="https://img.shields.io/badge/TypeScript-3178C6?style=flat-square&logo=typescript&logoColor=white"/> <img src="https://img.shields.io/badge/Tailwind_CSS-38B2AC?style=flat-square&logo=tailwindcss&logoColor=white"/> <img src="https://img.shields.io/badge/Supabase-3ECF8E?style=flat-square&logo=supabase&logoColor=black"/> <img src="https://img.shields.io/badge/Vercel-000000?style=flat-square&logo=vercel&logoColor=white"/>
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)